docs/contributors/contribute.md

```diff
@@ -24,77 +24,137 @@ After cloning the repository following the [github_workflow.md](github_workflow.
 make help
 ```
 
-### Setting Up the KinD Kubernetes Cluster
+### Quick Start
 
-For testing and development, we recommend using a KinD (Kubernetes in Docker) cluster.
+For the fastest way to get started, use the Makefile-driven workflow that sets up everything you need:
 
-1. Run the following command to create a KinD cluster:
+```sh
+make kind.setup
```

Comment:
Shall we use `make kind` to do the full setup and `make kind.cluster` to create the cluster specifically? This is to align with the current make target hierarchy.